addons/im_livechat/models/mail_channel.py

```diff
@@ -191,3 +193,108 @@ def _email_livechat_transcript(self, email):
             'body_html': mail_body,
         })
         mail.send()
+
+    def _get_channel_history(self):
+        """
+        Converting message body back to plaintext for correct data formatting in HTML field.
+        """
+        return ''.join(
+            '%s: %s<br/>' % (message.author_id.name or self.anonymous_name, html2plaintext(message.body))
+            for message in self.message_ids.sorted('id')
+        )
```

Comment:
Better to use Markupsafe when you build HTML

(`message.author_id.name` can contains weird characters)